linAssgn1

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+clear
+echo -n "Enter the Process Id to be killed : " 
+read prid
+if [ $prid -le 1 -o $prid -ge 32768 ]
+then
+echo "Invalid Range of Process Id"  
+elif
+ps -p $prid > /dev/null 2>&1 
+then
+kill -9 $prid
```

Comment:
http://stackoverflow.com/questions/9951556/why-number-9-in-kill-9-command-in-unix
http://en.wikipedia.org/wiki/Unix_signal#List_of_signals

Kindly read to get familiar with signalling